packages/deploy-script-support/test/unitTests/test-startInstance.js

```diff
@@ -47,10 +49,15 @@ test('startInstance', async t => {
     },
   };
 
+  /** @type {Petname=} */
```

Comment:
I don't know where it would go. I was burning down my problems pane.

Prefer that I take it out? would a separate commit in this PR with "merge" instead of squash suffice?

Reply:
separate commit would be nice. don't take it out, is my preference

Reply:
pulled non-run-protocol changes out to https://github.com/Agoric/agoric-sdk/pull/5440/commits/66fffd94c124a453a59ca8881981a0d47a976714 so I can squash this PR.